.github/workflows/unit-tests.yml

```diff
@@ -1,48 +1,13 @@
 name: unit-tests
 on: [push, pull_request]
 jobs:
-  gcc:
+  unit-tests:
```

Comment:
`unit-tests / unit-tests (:latest)`
/ME has temptation to point out we could do better, but really just temptation :)

Reply:
Oh right! Meh, why do we have to give a name to everything :-) This is hard, ideas appreciated!